docs/topics/getting-started-launchpad-update-booster-manual.adoc

```diff
@@ -48,14 +48,14 @@ NOTE: You can find your token in your {OpenShiftLocal} Web console. For example,
 $ mvn clean fabric8:deploy -Popenshift -DskipTests
 ----
 +
-This command uses the Fabric8 Maven Plugin to launch the S2I process on your {OpenShiftLocal} and to start the pod.
+This command uses the Fabric8 Maven Plugin to launch the link:{link-s2i-process}[S2I process] on your {OpenShiftLocal} and to start the pod.
```

Comment:
to create a BuildConfig on Openshift, where the input used is code compiled locally and pushed as binary stream to the platform (see https://docs.openshift.com/container-platform/3.6/dev_guide/builds/build_inputs.html#binary-source) and next to launch the S2I build on Openshift from the content uploaded and using the openjdk s2i image. If the build succeeds and a Docker image is pushed to the docker registry, then the DeploymentConfig is triggered to create and start the corresponding pod.

Reply:
@cmoulliard changed  link to reference the s2i process appendix  which summarizes this information.  Did same for the comment in docs/topics/launcher-review-update-booster-manual.adoc.